src/loaders/ImageLoader.js

```diff
@@ -15,7 +15,7 @@ ImageLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 	constructor: ImageLoader,
 
-	load: function ( url, onLoad, onProgress, onError ) {
+	load: function ( url, onLoad, onProgress, onError, priorityHint ) {
```

Comment:
Not a fan of adding `priorityHint` as a parameter to the `load()` method. If anything I would suggest adding a new method `setPriorityHint()`.

Reply:
Thanks for the feedback! I'll switch it around.